src/core/src/package_managers/AptitudePackageManager.py

```diff
@@ -521,17 +528,59 @@ def update_os_patch_configuration_sub_setting(self, patch_configuration_sub_sett
     # endregion
 
     # region Reboot Management
+    def do_processes_require_restart(self):
+        """ Fulfilling base class contract """
+        return False
+    # endregion Reboot Management
```

Comment:
nit: small suggestion - "# endregion Reboot Management" should be after def is_reboot_pending

Reply:
Will take this in next PR. as its minor.